src/mscorlib/shared/System/DateTimeOffset.cs

```diff
@@ -758,12 +792,25 @@ public static Boolean TryParse(String input, out DateTimeOffset result)
             return parsed;
         }
 
+        public static bool TryParse(ReadOnlySpan<char> input, out DateTimeOffset result)
+        {
+            bool parsed = DateTimeParse.TryParse(input, DateTimeFormatInfo.CurrentInfo, DateTimeStyles.None, out DateTime dateResult, out TimeSpan offset);
+            result = new DateTimeOffset(dateResult.Ticks, offset);
+            return parsed;
+        }
+
         public static Boolean TryParse(String input, IFormatProvider formatProvider, DateTimeStyles styles, out DateTimeOffset result)
         {
             styles = ValidateStyles(styles, nameof(styles));
```

Comment:
> styles = ValidateStyles(styles, nameof(styles)); [](start = 11, length = 49)

Interesting we throw on invalid styles but we don't throw when input is null. I know you are not changing any behavior but this behavior surprised me :-)